### DISCUSSION TITLE
使用CloudFlare CNAME 接入后，无法正常使用。

### DISCUSSION THREAD

**Discussion by Troray** [1Panel-dev/1Panel#5511] [Q&A]
按该文方法: https://blog.csdn.net/qq_31066285/article/details/133690049 接入后，无法正常打开用1Panel搭建的网站。
我这里在用文字简要说一下过程。
两个域名
一个在cloudflard上：`cf.1234.xyz` ,该域名解析到1Panel所在服务器的网站，并且开启cf中的“小云朵”。
一个在huawei云解析上: `hw.abcd.com`

1.`cf.1234.xyz` ,该域名解析到1Panel所在服务器的网站，并且开启cf中的“小云朵”。
2. 到`ssl/tls`--`自定义主机名` 中添加`回退源` `cf.1234.xyz`。
3. 添加自定义主机名：`hw.abcd.com`
4. 到huawei云解析添加对应的CNAME记录。
5. 等待CloudFlare上回源和自定义主机名生效即完成。

现在出现的情况是，以上操作生效后，访问`hw.abcd.com`出现如下图。。但访问源域名`cf.1234.xyz` 无任何问题
![Snipaste_2024-06-19_19-21-27](https://github.com/1Panel-dev/1Panel/assets/16841721/9fe2c22c-2e7a-4d45-80a7-88a705d55119)

F12：
```
请求 URL:https://hw.abcd.com/
请求方法:GET
状态代码:304 Not Modified
```
---

**同时，按以上完全一致的方法，在另外一台服务器上，仅使用nginx部署的一个网站，则能正常访问。**

**Comment by Troray** [comment]
晚上又检查了一下，发现出现的错误和直接点击OpenResty中的端口一样，处于无绑定解析的状态。
![Snipaste_2024-06-19_23-38-55](https://github.com/1Panel-dev/1Panel/assets/16841721/aa687b2a-af94-4aea-b2c5-99a2470cebb7)

**Comment by okxlin** [comment]
hw.abcd.com
要再解析到cf

解析到cf ip，或者cname到cf优选站点

才能保证cf正确回源

**Comment by miaowmint** [comment]
你的源站需要同时绑 hw.abcd.com 和 cf.1234.xyz 这两个域名，你确定你都绑了吗

**Comment by b1ghawk** [comment]
你的源站需要同时绑 hw.abcd.com 和 cf.1234.xyz 这两个域名。
此外也要让hw.abcd.com A记录到静态优选的cf ip，或者 CNAME到某些能够解析到动态优选cf ip的域名。
